docs/tutorial.md

```diff
@@ -69,7 +69,6 @@ dumb questions with patience
 * It's a command-line utility. You pretty much have to be a programmer/devops person to 
 use Gutenberg. Downloading a single file and copying it to a directory
 is arguably a little harder than harder than downloading, then clicking an installer
-* A tiny Gutenberg site is a little more complicated than a tiny HTML site, due to its configuration requirements
```

Comment:
you can keep this but the minimum config required is 1 line, the base url. Unless by configuration you mean having to get the binary in which case, fair enough!